scipy/stats/tests/test_distributions.py

```diff
@@ -3420,6 +3420,29 @@ def test_isf(self):
                           330.6557590436547, atol=1e-13)
 
 
+class TestDgamma:
+
+    @pytest.mark.parametrize("a, ref",
+        [(1.5, 2.0541199559354118267922018421759071394202272087878735),
+        (1.3, 1.9357296377121247644586320317632592032856887302320274),
+        (1.1, 1.7856502333412133408733521579693880954302277243835155)])
+    def test_entropy(self, a, ref):
+        #The reference values were calculated with mpmath:
+        #def entropy_dgamma(a):
+        #   def pdf(x):
+        #       A = 1 / (2 * mp.gamma(a))
+        #       B = mp.fabs(x) ** (a - 1)
+        #       C = mp.exp(-mp.fabs(x))
+        #       h = A * B * C
+        #       return h
+        #
+        #   return -mp.quad(lambda t: pdf(t) * mp.log(pdf(t)), [-mp.inf, mp.inf])
+        assert_allclose(stats.dgamma._entropy(a), ref)
+
+    def test_entropy_overflow(self):
+        assert np.isfinite(stats.dgamma.entropy(1e100))
+        assert np.isfinite(stats.dgamma.entropy(1e-100))
```

Comment:
```suggestion
```

These should be removed in favor of actual tests against the result computed with mpmath for extreme values for `m`  (see above).